src/utils/index.js

```diff
@@ -2,14 +2,14 @@
 
 import _ from 'lodash'
 
-exports.labeler = require('./labeler')
-exports.canvasUtils = require('./canvasUtils')
-exports.dataGeneration = require('./dataGeneration')
-exports.imageRender = require('./imageRender')
-exports.path2DUtils = require('./path2DUtils')
-exports.rectUtils = require('./rectUtils')
-exports.windowUtils = require('./windowUtils')
-exports.stateHOC = require('./stateHOC')
+export * as labeler from './labeler'
```

Comment:
Similarly, if you are trying to re-export a single default export, you want `export {default as labeler} from './labeler'`, etc.

This will depend on whether the referenced module has named exports or a default export.